src/components/ImageView/LabelOnRegion.js

```diff
@@ -103,8 +103,6 @@ const LabelOnPolygon = observer(({ item }) => {
           shadowBlur={0}
         />
       )}
-      {/* TODO this is a problem here, it takes states[0] but nothing
-       * guarantees that this state is a lable one */}
```

Comment:
Good point. basically, the controls tags need to implement the interface and here we would just look up the state that provide that